ietf-layer0-types.yang

```diff
@@ -756,31 +479,43 @@ module ietf-layer0-types {
           range "1..max";
         }
         description
-          "Maximum slot width is calculated by:
+          "Slot width range: two multipliers of the slot width ,
+           granularity, each indicating the minimal and maximal slot 
+           width supported by a port, respectively.
+
+           Maximum slot width is calculated by:
              Maximum slot width (GHz) = 
                max-slot-width-factor * slot-width-granularity";
         reference
           "RFC8363: GMPLS OSPF-TE Extensions in Support of Flexi-Grid
           Dense Wavelength Division Multiplexing (DWDM) Networks";
       }
-
     }
   }
 
   grouping flexi-grid-label-step {
     description "Label step information for flexi-grid";
-    leaf flex {
+    leaf flexi-grid-channel-spacing {
       type identityref {
         base flexi-ch-spc-type;
       }
       default flexi-ch-spc-6p25ghz;
       description
         "Label-step is the nominal central frequency
          granularity (GHz), e.g., 6.25 GHz";
+      reference
+         "RFC7699: Generalized Labels for the Flexi-Grid in
+          Lambda Switch Capable (LSC) Label Switching Routers";
+    } 
+    leaf flexi-n-step {
+      type uint8;
+      description
+        "The restriction of flexi-n must be certain multipliers 
+          of channel-spacing. This is called the central frequency
+          granularity in RFC8363. ";
```

Comment:
Could you re-phrase the description of this leaf not sure to have understood what you mean here ? It is mentioned central frequency granularity that is a definition specific for RFC 8363. This RFC says that for any "definitions" the reference IETF is RFC 7698. I do not like central frequency granularity and we need to be more clear in the description.

Reply:
@sergiobelotti : what about replacing the description with this one?

```
This attribute defines the granularity of supported values for the nominal central frequency as a multiplier of the channel-spacing.

For example, given a grid with a nominal central frequency granularity of 6.25 GHz, the granularity of the supported values of the nominal central frequency could be 12.5 GHz. In this case, the values of flexi-n should be even and this constraints is reported by setting the flexi-n-step to 2.

This attribute is also known as central frequency granularity in RFC8363.
```